maskrcnn_benchmark/engine/bbox_aug.py

```diff
@@ -0,0 +1,118 @@
+import torch
+import torchvision.transforms as TT
+
+from maskrcnn_benchmark.config import cfg
```

Comment:
using the config globally in the library is something I've tried to avoid, but let's merge this as is and maybe modify this later on